### PR TITLE
replaced gca(projection="3d") with plt.axes(projection="3d") 

### PR DESCRIPTION
**Pull request by stakaz**
Replaced gca(projection="3d") with plt.axes(projection="3d")  as is needed for matplotlib 3.4.2 and later.

The `gca()` command does not acept keyword arguments anymore.

### CODE REVIEW

**Comment by PaulSoderlind** [comment]
Maybe I typed in my comment at the wrong place. Here we go again:

Thanks, but I think this might be a problem when you do 3D subplots and even more so on matplotlib 3.3. What seems to work with both matplotlib 3.3&3.4 and also with both standard plots&subplots is this:

`PyPlot.version <= v"3.4" ? ax = gca(projection="3d") : ax = plt.subplot(projection="3d")`

**Comment by stakaz** [comment]
Ok, I changed it to your proposed version.

File: src/plot3d.jl [hunk 1]
```diff
@@ -58,7 +58,7 @@ for f in mplot3d_funcs
     fs = string(f)
     @eval @doc LazyHelp(axes3D,"Axes3D", $fs) function $f(args...; kws...)
         using3D() # make sure mplot3d is loaded
-        ax = gca(projection="3d")
+					PyPlot.version <= v"3.4" ? ax = gca(projection="3d") : ax = plt.subplot(projection="3d")
```
Comment (stevengj) [inline src/plot3d.jl:61]: Probably
```jl
ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```

Comment (PaulSoderlind) [reply src/plot3d.jl:61]: OK, very good. (I now see how the plt."" is used.)

Comment (stakaz) [reply src/plot3d.jl]: Included this in the PR.

File: src/plot3d.jl [hunk 2]
```diff
@@ -58,7 +58,7 @@ for f in mplot3d_funcs
     fs = string(f)
     @eval @doc LazyHelp(axes3D,"Axes3D", $fs) function $f(args...; kws...)
         using3D() # make sure mplot3d is loaded
-        ax = gca(projection="3d")
+					ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```
Comment (stevengj) [inline src/plot3d.jl:61]:
```suggestion
        ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```

File: src/plot3d.jl [hunk 3]
```diff
@@ -74,7 +74,7 @@ for f in zlabel_funcs
     fs = string("set_", f)
     @eval @doc LazyHelp(axes3D,"Axes3D", $fs) function $f(args...; kws...)
         using3D() # make sure mplot3d is loaded
-        ax = gca(projection="3d")
+					ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```
Comment (stevengj) [inline src/plot3d.jl:77]:
```suggestion
        ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```

File: src/plot3d.jl (hunk 2 above)
Comment (stevengj) [reply src/plot3d.jl:61]: (Somehow your indenting is messed up.  Maybe you used tabs?)

File: src/plot3d.jl (hunk 3 above)
Comment (stakaz) [reply src/plot3d.jl]: Yes, exactly, it is now corrected.

File: src/plot3d.jl [hunk 4]
```diff
@@ -58,7 +58,7 @@ for f in mplot3d_funcs
     fs = string(f)
     @eval @doc LazyHelp(axes3D,"Axes3D", $fs) function $f(args...; kws...)
         using3D() # make sure mplot3d is loaded
-        ax = gca(projection="3d")
+          ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```
Comment (stevengj) [inline src/plot3d.jl:61]:
```suggestion
        ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```
The indenting is still wrong

File: src/plot3d.jl [hunk 5]
```diff
@@ -74,7 +74,7 @@ for f in zlabel_funcs
     fs = string("set_", f)
     @eval @doc LazyHelp(axes3D,"Axes3D", $fs) function $f(args...; kws...)
         using3D() # make sure mplot3d is loaded
-        ax = gca(projection="3d")
+          ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```
Comment (stevengj) [inline src/plot3d.jl:77]:
```suggestion
        ax = PyPlot.version <= v"3.4" ? gca(projection="3d") : plt."subplot"(projection="3d")
```

**Comment by stakaz** [comment]
Thanks, sorry for this small indentation error ;)

**Pull request merged**